### DISCUSSION TITLE
kolla upgrades with OSISM only per service?

### DISCUSSION THREAD

**Discussion by Nils98Ar** [osism/issues#542] [Q&A] [answered]
It seems that plain kolla-ansible supports upgrading all services at once with:
```
kolla-ansible pull
kolla-ansible prechecks
kolla-ansible upgrade
```
(https://docs.openstack.org/kolla-ansible/latest/user/operating-kolla.html#perform-the-upgrade)

@berendt Am I Right that this is not possible/intended in OSISM? What is the reason for this?

**Comment by berendt** [comment, marked as answer]
This is on purpose as we only use Kolla as a base to deploy the services and don't use the kolla-ansible CLI etc. We are working on the superior orchestrator with Zuul so that it is no longer necessary to control the services individually.

**Comment by Nils98Ar** [comment]
Thank you!

Addition: is it still necessary to run the nova upgrade on control and compute nodes separately after each other?

**Reply by berendt** [reply]
No, this is handled in the playbook used for Nova.
